BP/history/characters/estonian.txt

```diff
@@ -1,8 +1,8 @@
-109717 = { 
```

Comment:
Why the changed name?

Reply:
In vanilla, they changed it from ugricbaltic CK2 to Estonian CK3 so I could've gone either way and just went with vanilla for best compatibility.

Reply:
I realise that is just a code name change in vanilla, that makes sense.